test/plugin/containers/jvm-container/pom.xml

```diff
@@ -32,8 +32,63 @@
     <artifactId>jvm-container</artifactId>
     <name>SkyWalking JVM Runner Image</name>
 
-    <properties>
-        <skip.docker>false</skip.docker>
-        <docker.image.name>skywalking/agent-test-jvm</docker.image.name>
-    </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>0.33.0</version>
+                <configuration>
+                    <images>
+                        <image>
+                            <name>skywalking/agent-test-jvm:${BUILD_NO}</name>
+                            <build>
+                                <from>openjdk:8</from>
+                                <workdir>/usr/local/skywalking/tools</workdir>
+                                <assembly>
+                                    <mode>dir</mode>
+                                    <targetDir>/usr/local/skywalking</targetDir>
+                                    <descriptor>assembly.xml</descriptor>
+                                </assembly>
+                                <runCmds>
+                                    <run>chmod +x /usr/local/skywalking/run.sh</run>
+                                    <run>tar -xvf ../tools/skywalking-mock-collector.tar.gz -C ../tools</run>
+                                </runCmds>
+                                <workdir>/usr/local/skywalking/scenario</workdir>
+                                <cmd>/usr/local/skywalking/run.sh</cmd>
+                            </build>
+                        </image>
+                        <image>
+                            <name>skywalking/agent-test-jvm:jdk14-${BUILD_NO}</name>
```

Comment:
Why I can't find the place to declare using this image? Could you point out?

Reply:
It just is used to jdk14 case. And now, I have not enabled it yet.

Reply:
I have a big change in building docker image, so I push it for ensuring that is ok.

Reply:
Got it. I was trying to find, how to activate the jdk14 container, notice it is an option only.